### ISSUE TITLE
loss is extremely large（about 100000）when training on V100

### ISSUE THREAD

**Issue by edwardning** [zylo117/Yet-Another-EfficientDet-Pytorch#379]
Hi！
When I was training on my own laptop (GPU is mx250, for test), everything is OK, and the loss was generally less than 10. But when I was training on a Tesla V100, the loss increased sharply to about 100000. Have you ever encountered any similar situation? Do you have any suggestions?
Thanks！
-----------------------------------------------------------------------------------------------------------
Step: 0. Epoch: 0/500. Iteration: 1/93. Cls loss: 245419.45312. Reg loss: 0.15489. Total loss: 245419.60938
Step: 1. Epoch: 0/500. Iteration: 2/93. Cls loss: 201541.98438. Reg loss: 0.15606. Total loss: 201542.14062
Step: 2. Epoch: 0/500. Iteration: 3/93. Cls loss: 209735.20312. Reg loss: 0.15990. Total loss: 209735.35938
Step: 3. Epoch: 0/500. Iteration: 4/93. Cls loss: 164546.79688. Reg loss: 0.16808. Total loss: 164546.96875
Step: 4. Epoch: 0/500. Iteration: 5/93. Cls loss: 328964.03125. Reg loss: 0.14675. Total loss: 328964.18750

**Comment by zylo117** [comment]
Same command line?
What about environments?

**Comment by edwardning** [comment]
> Same command line?
> What about environments?

Thank you! 
Found the problem, i. e.  I forget to load pre_trained weights when training on my laptop. 

But I still confused, why the total loss (training with pre_trained weights) is such bigger than that training without pre_trained weights?

And it seems that the total loss is mainly from cls loss.

**Comment by zylo117** [comment]
In general, cls head is harder to train than reg head, esecially when you are training on small datasets.

**Comment by edwardning** [comment]
> neral, cls head is harder to train than reg head, esecially when you are traini

Thanks for your reply! Got it.
